app/assets/javascripts/modules/want_to_talk.js

```diff
@@ -0,0 +1,244 @@
+define(
+  ['jquery', 'common'],
+  function($, MAS) {
+    'use strict';
+
+    var COMPONENT_SELECTOR = '.want-to-talk',
+        SIDE_PARENT_SELECTOR = '.l-article-3col-right',
+        INLINE_PARENT_SELECTOR = '[role="main"] > p:first',
+        MAIN_CONTENT_SELECTOR = '.l-article-3col-main',
+        BODY_SELECTOR = '.want-to-talk__body',
+        INLINE_CLASS = 'want-to-talk--inline',
+        SIDEBAR_CLASS = 'want-to-talk--sidebar',
+        FIXED_CLASS = 'want-to-talk--fixed',
+        BOTTOM_CLASS = 'want-to-talk--bottom',
+        OPEN_CLASS = 'want-to-talk--open',
+        INITIALISED_CLASS = 'want-to-talk--initialised',
+        TRIGGER_SELECTOR = '.want-to-talk__trigger__button, .want-to-talk__title',
+        LABEL_SELECTOR = '.want-to-talk__trigger__button__text',
+        TEL_LINK_SELECTOR = '.want-to-talk__body__phone',
+        TRACK_CATEGORY = 'TriagePhase1',
+        MEDIUM_MIN_WIDTH = 720,
```

Comment:
This should be defined in the CSS to avoid divergence. You can check that a media query is activated by querying an invisible `:after` element. [Jeremy Keith explains it here](https://adactio.com/journal/5429) (at the bottom of page).